README.md

```diff
@@ -43,18 +43,22 @@ More information in the grafana configuration documentation: http://docs.grafana
 ## Grafana container with persistent storage (recommended)
 
 ```
-# create /var/lib/grafana as persistent volume storage
-docker run -d -v /var/lib/grafana --name grafana-storage busybox:latest
+# create a persistent volume for your data in /var/lib/grafana (database and plugins)
+docker volume create grafana-storage
 
 # start grafana
 docker run \
   -d \
   -p 3000:3000 \
   --name=grafana \
-  --volumes-from grafana-storage \
+  -v grafana-storage:/var/lib/grafana \
```

Comment:
should probably have `:z` as an argument as well, in case of SELinux?

Reply:
Never seen that one before. Will look into it. Thanks for reviewing the container, much appreciated.